### PR TITLE
[docs] Clarify using the ClusterConfiguration and the <provider>ClusterConfiguration parameters

### PR DESCRIPTION
**Pull request by z9r5**
## Description
Clarify using the ClusterConfiguration and the <provider>ClusterConfiguration parameters.

Close: #2957

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [x] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries
```changes
section: doc
type: fix
summary: Clarify using the ClusterConfiguration and the <provider>ClusterConfiguration parameters.
impact_level: low
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

File: docs/documentation/pages/DECKHOUSE-FAQ.md [hunk 1]
```diff
@@ -251,24 +251,19 @@ To switch the Deckhouse cluster to using a third-party registry, follow these st
...
 
-The general cluster parameters are stored in the `ClusterConfiguration` structure. It contains parameters such as:
-
-- cluster domain: `clusterDomain`;
-- CRI used in the cluster: `defaultCRI`;
-- Kubernetes control plane version: `kubernetesVersion`;
-- cluster type (Static, Cloud): `clusterType`;
-- address space of the cluster's Pods: `podSubnetCIDR`;
-- address space of the cluster's services: `serviceSubnetCIDR` etc.
+The general cluster parameters are stored in the [ClusterConfiguration](installing/configuration.html#clusterconfiguration) structure. It contains parameters such as:
```
Comment (yalosev) [inline docs/documentation/pages/DECKHOUSE-FAQ.md:254]: Some cutted phrase here. `It contains parameters such as:`

Comment (z9r5) [inline docs/documentation/pages/DECKHOUSE-FAQ.md:254]:
```suggestion
The general cluster parameters are stored in the [ClusterConfiguration](installing/configuration.html#clusterconfiguration) structure.
```

Comment (z9r5) [reply docs/documentation/pages/DECKHOUSE-FAQ.md]: Fixed. Thanks.

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Review by yalosev — APPROVED** [review]

**Pull request merged**